src/components/colorbar/defaults.js

```diff
@@ -128,6 +128,9 @@ module.exports = function colorbarDefaults(containerIn, containerOut, layout) {
         weight: font.weight,
         style: font.style,
         variant: font.variant,
+        capitalize: font.capitalize,
+        striding: font.striding,
+        shadow: font.shadow,
```

Comment:
seems like this should be inverted now, something like:
```js
var dfltTitleFont = Lib.extendFlat({}, font, {
    family: tickFont.family,
    size: Lib.bigFont(tickFont.size)
});
```

Reply:
Good point. Addressed in [d7c6f8f](https://github.com/plotly/plotly.js/pull/6983/commits/d7c6f8f2278961c5372f367b86e1ae29ea7155d3).
Thank you!